server/sonar-web/src/main/less/components/columns.less

```diff
@@ -32,6 +32,13 @@
   overflow: visible !important;
 }
 
+.column-full {
```

Comment:
Ok, you want me to move all of them ?
I did that because I thought those class might be useful in other part than the billing plugin.